pkg/update/check.go

```diff
@@ -24,6 +25,9 @@ func Check(ctx context.Context, currentVersion string, cliVersioner Versioner) (
 		return current, latest, false, fmt.Errorf("error fetching latest version: %w", err)
 	}
 
+	asset := fmt.Sprintf("%s_v%s_%s-%s.tar.gz", cliVersioner.Binary(), latest, runtime.GOOS, runtime.GOARCH)
```

Comment:
This template string is used in a couple of places across the code base, maybe consider moving it to a package level constant.

Reply:
👍🏻